pipeline-model-definition/src/test/resources/basic/stages100WithOutsideVarAndFunc.groovy

```diff
@@ -23,6 +23,7 @@
  */
 
 def someVar = "Hi there"
+def (firstVar, secondVar) = [1, 2]
```

Comment:
Looks like this does work in Pipeline as of [JENKINS-45575](https://issues.jenkins.io/browse/JENKINS-45575). Maybe I was thinking of [JENKINS-55001](https://issues.jenkins.io/browse/JENKINS-55001), which apparently only matters for sandboxed scripts using `script-security` but not `workflow-cps`.